Description: Changes QGraphicsVideoItem to use xvoverlay rendering
instead of gles2 painting. Reason for this is that current implementation
of gles2 painting(QVideoSurfaceGlslPainter) doesn't support UYVY and YUY2
pixel formats. It is also very slow (only 13 FPS) when tested on n900.
Qml camera component uses QGraphicsVideoItem to render view finder stream.
Author: Joonas Tanskanen <joonas.tanskanen@sasken.com>
--- a/plugins/multimedia/gstreamer/camerabin/camerabinservice.cpp
+++ b/plugins/multimedia/gstreamer/camerabin/camerabinservice.cpp
@@ -119,9 +119,10 @@ CameraBinService::CameraBinService(const
 #endif
 
 
-#ifdef Q_WS_MAEMO_6
+#if defined(Q_WS_MAEMO_6)
         m_videoWindow = new QGstreamerVideoWindow(this, "omapxvsink");
-        //m_videoWindow = new QGstreamerVideoWindow(this);
+#elif defined(Q_WS_MEEGO)
+        m_videoWindow = new QGstreamerVideoWindow(this);
 #else
         m_videoWindow = new QGstreamerVideoOverlay(this);
 #endif
--- a/src/multimedia/multimedia.pro
+++ b/src/multimedia/multimedia.pro
@@ -192,8 +192,11 @@ symbian {
         SOURCES += qgraphicsvideoitem_overlay.cpp
     }
 }
+meego {
+    SOURCES += qgraphicsvideoitem_overlay.cpp
+}
 
-!maemo*:!symbian {
+!meego:!maemo*:!symbian {
     SOURCES += qgraphicsvideoitem.cpp
 }
 
